test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -201,14 +201,20 @@ public HapiApiSpec creationHappyPath() {
 								.treasury(TOKEN_TREASURY)
 								.autoRenewAccount("autoRenewAccount")
 								.autoRenewPeriod(A_HUNDRED_SECONDS)
+								.maxSupply(1000)
```

Comment:
Specifying only `maxSupply` without `supplyType` `FINITE` should result in an error